build/cni/Makefile

```diff
@@ -22,13 +22,14 @@ CNI_RELEASE?=07a8a28637e97b22eb8dfe710eeae1344f69d16e
 CNI_TARBALL=cni-$(ARCH)-$(CNI_RELEASE).tar.gz
 CUR_DIR=$(shell pwd)
 OUTPUT_DIR=$(CUR_DIR)/output
+GOLANG_VERSION=1.7
```

Comment:
`docker build` has `--pull`, but for `run` you have to `docker pull` then `docker run` - see https://github.com/docker/docker/issues/13331.